Antidote/ActiveCallViewController.m

```diff
@@ -319,6 +349,71 @@ - (void)showCallPausedByFriend
     [self createFriendAvatar];
 }
 
+- (void)setVideoView:(UIView *)videoView
+{
+    _videoView = videoView;
+
+    if (self.videoView) {
+
+        if (self.previewView) {
+            [self.view insertSubview:self.videoView belowSubview:self.previewView];
+        }
+        else {
+            [self.view addSubview:self.videoView];
+        }
+
+        [self.videoView makeConstraints:^(MASConstraintMaker *make) {
+            make.bottom.equalTo(self.endCallButton.top).with.offset(-kIndent);
+            make.top.equalTo(self.topViewContainer.bottom).with.offset(kIndent);
+            make.left.equalTo(self.view);
+            make.right.equalTo(self.view);
+        }];
+
+        [self updateCallsControlViewCondensed];
+    }
+    else {
+        [self.videoView removeFromSuperview];
```

Comment:
If there is no `self.videoView` there is nothing to remove from superview. You have to

```objc
- (void)setVideoView:(UIView *)videoView
{
    // sanity check
    if (_videoView == videoView) {
        return;
    }

    // check if there was old feed viev
    if (_videoView) {
        [_viewView removeFromSuperview];
        ...
    }

    // only then reassign it
    _videoView = videoView;

    // and if there is a new one, do stuff you do with it
    if (! videoView) {
        return;
    }
    
    // do stuff
    ...
}
```